docs/journeys/embedding.md

```diff
@@ -0,0 +1,234 @@
+---
+sidebar_position: 3
+---
+
+# Embedding Journeys
+## Introduction
+What is the embed script?
+The embed script provides a simple, yet flexible way to place a journey into your website. It is a wrapper around the created iframe, where the journeys operate in. 
+
+It’s essentially initializing the journey in form of an iframe, based on the provided journeyId and exposes and interface to interact with it.
+epilot Journeys can easily be embedded into your website. All you have to do is copy the embed script of your journey to the appropriate place on your website and modify it slightly if necessary.
+
+### Aim of the article
+This article aims to guide you through the different options available to embed a Journey into your web page. 
```

Comment:
```suggestion

Journeys can easily be embedded on any website by placing an embed script tag provided by epilot on your website, and then optionally customizing it for advanced use.

## Embed Script

An embed script is a simple way to publish an epilot journey on your website. The script loads the journey using a Journey ID and a secure token to embed the journey securely on your website.

This page guides you through the different options available to embed a Journey into your web page. 
```

Reply:
Just slightly modified the style of the introduction to better match the rest of docs.epilot.io